### PR TITLE
add code, notes, and plots for popular controlled access datasets

### PR DESCRIPTION
**Pull request by kweav**
This PR specifically considers the question of which controlled access datasets respondents do use or would like to use if they were available on AnVIL. There is code in the clean data section that adds 3 columns used as simplified status flag for clinical, human genomic, and non-human genomic research experience. There is then code in the insights section that  prepares the data for plotting, specifically cleaning the dataset responses, subsetting based off of the simplified experience flags, and annotating based on whether the datasets are available on the AnVIL or not. Finally, 4 plots are made. One with all the survey responses, one looking at just responses from those experienced with clinical research, another looking at just responses from those experienced with human genomic research, and finally one looking at just responses from those experienced with non-human genomic research.

Waiting on verification concerning which datasets are currently available on the AnVIL or not. So code defining the onAnVILDF may change. Also likely to add code later to more directly see if there are differences in popular datasets for those experienced in human genomics vs human clinical.

### CODE REVIEW

File: sota24analysis.Rmd [hunk 1]
```diff
@@ -753,6 +789,234 @@ ggplot(countdf, aes(fill=rank, y=Feature, x=n)) +
...
+  --> We recoded this to be an "Other"
+* "TCGA"
+  --> This response was left as is since there is a controlled access tier.
+
+</details>
+
+```{r}
+prep_df_whichData <- function(subset_df){
+  
+  onAnVILDF <- data.frame(whichControlledAccess = c( #checking on this
```
Comment (avahoffman) [inline sota24analysis.Rmd:855]: Any reason not to have this in a txt like the institution codebook?

File: sota24analysis.Rmd [hunk 2]
```diff
@@ -753,6 +789,234 @@ ggplot(countdf, aes(fill=rank, y=Feature, x=n)) +
...
+whichDataClinicalSubset <- resultsTidy %>%
+  filter(clinicalFlag == TRUE) %>%
+  prep_df_whichData()
+
+whichDataHumanGenomicSubset <- resultsTidy %>%
+  filter(humanGenomicFlag == TRUE) %>%
+  prep_df_whichData()
+
+whichDataNonHumanGenomicSubset <- resultsTidy %>%
+  filter(nonHumanGenomicFlag == TRUE) %>%
```
Comment (avahoffman) [inline sota24analysis.Rmd:942]: Is there a large % that have both human and nonhuman genomic expertise? Just curious what that table looks like.

File: sota24analysis.Rmd [hunk 3]
```diff
@@ -753,6 +789,234 @@ ggplot(countdf, aes(fill=rank, y=Feature, x=n)) +
...
+
+</details>
+
+```{r}
+
+plot_which_data <- function(inputToPlotDF, subtitle = NULL){
+toreturnplot <- ggplot(inputToPlotDF, aes(x = reorder(whichControlledAccess, -count), y = count, fill = AnVIL_Availability)) +
+  geom_bar(stat="identity") + 
+  theme_classic() + theme(panel.background = element_blank(), panel.grid = element_blank()) +
+  theme(axis.text.x = element_text(angle=45, hjust=1)) +
```
Comment (avahoffman) [inline sota24analysis.Rmd:974]: could save space with inset legend, ie
```
+ theme(legend.position = c(0.8, 0.8))
```

**Review by avahoffman — APPROVED** [review]
Looks good!

File: sota24analysis.Rmd (hunk 1 above)
Comment (kweav) [reply sota24analysis.Rmd:855]: Great idea! I can do this

File: sota24analysis.Rmd (hunk 2 above)
Comment (kweav) [reply sota24analysis.Rmd:942]: Great question -- will look into this

File: sota24analysis.Rmd (hunk 3 above)
Comment (kweav) [reply sota24analysis.Rmd:974]: Will try this as well! Thanks!

**Pull request merged**